codebuild_specs/run_coverage_linux.yml

```diff
@@ -0,0 +1,30 @@
+version: 0.2
+env:
+  shell: bash
+  variables:
+    CLI_REGION: us-east-1
+    E2E_TEST_COVERAGE_DIR: node_v8_coverage
+  secrets-manager:
+    CODECOV_TOKEN: arn:aws:secretsmanager:us-east-1:052112422291:secret:testing/codecov/token-OB8isC
```

Comment:
https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#secrets-manager-build-spec

Do we need the full ARN here? Is this secret stored in a different account? @awsluja any insight here? Putting a full ARN here feels wrong.